### ISSUE TITLE
Would you mind share the script used for finetune the model?

### ISSUE THREAD

**Issue by kexul** [mrzjy/GenshinCLIP#1]
Would you mind share the script or a notebook for finetune the siglip model? There is very little relevant information online.

**Comment by mrzjy** [comment]
Sure, you can refer to [this guide](https://github.com/mlfoundations/open_clip/discussions/812).
As for applying the siglip loss function, just add "--siglip" flag (See [ref](https://github.com/mlfoundations/open_clip/blob/main/src/open_clip_train/params.py#L449) for this argument)

**Comment by kexul** [comment]
Thanks a lot!
